packages/helper-settings/SettingsForm.js

```diff
@@ -53,8 +58,11 @@ export default class Form extends Component {
     if (!response.login) {
       this.setState({
         tokenLoaded: false,
-        errorMessage: response.message || 'Something went wrong',
       });
+      this.tokenInputEl.setCustomValidity(
+        response.message || 'Something went wrong',
```

Comment:
Should the default message be more specific? Like “The token could not be validated”

Reply:
Good point. Also the API response is not very descriptive `Bad credentials` in my opinion. Will change it later